### PR TITLE
Remove cacert platform

### PR DESCRIPTION
**Pull request by jeremyn**
This pull request removes the `cacert` `platform`. See https://github.com/EFForg/https-everywhere/pull/14833#issuecomment-371172946.

Ping @Hainish .

### CODE REVIEW

File: chromium/rules.js [hunk 1]
```diff
@@ -255,7 +255,7 @@ RuleSets.prototype = {
     }
 
     // If a ruleset declares a platform, and we don't match it, treat it as
-    // off-by-default. In practice, this excludes "mixedcontent" & "cacert" rules.
+    // off-by-default. In practice, this excludes "mixedcontent" rules.
```
Comment (Hainish) [inline chromium/rules.js]: There is another reference to this in https://github.com/EFForg/https-everywhere/pull/14875/files#diff-8809792043b9914a5eea37df1e10a564R463

**Comment by Hainish** [comment]
Otherwise looks good.  Please rebase from master and I'll merge.

**Comment by jeremyn** [comment]
Done.

**Comment by Hainish** [comment]
Thank you!

**Comment by Bisaloo** [comment]
Should we expect any effect of this on downstream dependencies? 

I expect some pieces of code to become useless (such as [this test in Tenta Browser HTTPS Everywhere library](https://github.com/tenta-browser/go-https-everywhere/commit/8211474a484b2aac01810ffa14daa60c10b002b4)) but as far as I can tell, nothing should break because of this.

Can you think of something else?

**Comment by jeremyn** [comment]
Thanks @Hainish .

@Bisaloo To your point about dependencies: this would be a good thing to add to the next release notes.

**Pull request merged**